public/app/core/utils/markdown.ts

```diff
@@ -0,0 +1,19 @@
+import config from 'app/core/config';
+import marked from 'marked';
+
+const options: any = {
```

Comment:
I see `MarkedOptions` type here https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/marked/index.d.ts#L252 - can we use it?